examples/example-two/netlify.yml

```diff
@@ -13,22 +13,23 @@ build:
 
 # Build Plugins
 plugins:
-  - type: ./plugins/plugin-one
-  - type: ./plugins/plugin-two
-  - type: ./plugins/dynamic
-    config:
-      hook: init
-  - type: '@netlify/plugin-no-more-404'
-    config:
-      on404: 'error'
-      blankSlate: false
-  - type: '@netlify/plugin-lighthouse'
-    enabled: ${env:LIGHTHOUSE_ENABLED, false}
-    config:
-      currentVersion: 0.0.4
-      compareWithVersion: 0.0.3
-  - type: '@netlify/plugin-axe'
-    enabled: false
-    config:
-      # https://github.com/dequelabs/axe-cli#running-specific-rules
-      axeFlags: '--rules color-contrast,html-has-lang'
+  - type: '@netlify/plugin-search-index'
+  # - type: ./plugins/plugin-one
+  # - type: ./plugins/plugin-two
+  # - type: ./plugins/dynamic
+  #   config:
+  #     hook: init
+  # - type: '@netlify/plugin-no-more-404'
+  #   config:
+  #     on404: 'error'
+  #     blankSlate: false
+  # - type: '@netlify/plugin-lighthouse'
+  #   enabled: ${env:LIGHTHOUSE_ENABLED, false}
+  #   config:
+  #     currentVersion: 0.0.4
+  #     compareWithVersion: 0.0.3
+  # - type: '@netlify/plugin-axe'
+  #   enabled: false
+  #   config:
+  #     # https://github.com/dequelabs/axe-cli#running-specific-rules
+  #     axeFlags: '--rules color-contrast,html-has-lang'
```

Comment:
Would it be possible to re-enable those plugins? I am guessing you might have commenting them out for debugging.